Sources/LanguageServerProtocol/Requests/OpenInterfaceRequest.swift

```diff
@@ -10,40 +10,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// Request a generated interface of a module to display in the IDE.
-/// **(LSP Extension)**
-public struct OpenGeneratedInterfaceRequest: TextDocumentRequest, Hashable {
-  public static let method: String = "textDocument/openInterface"
-  public typealias Response = GeneratedInterfaceDetails?
-
-  /// The document whose compiler arguments should be used to generate the interface.
-  public var textDocument: TextDocumentIdentifier
-
-  /// The module to generate an index for.
-  public var moduleName: String
-
-  /// The module group name.
-  public var groupName: String?
-
-  /// The symbol USR to search for in the generated module interface.
-  public var symbolUSR: String?
-
-  public init(textDocument: TextDocumentIdentifier, name: String, groupName: String?, symbolUSR: String?) {
-    self.textDocument = textDocument
-    self.symbolUSR = symbolUSR
-    self.moduleName = name
-    self.groupName = groupName
-  }
-
-  /// Name of interface module name with group names appended
-  public var name: String {
-    if let groupName {
-      return "\(self.moduleName).\(groupName.replacing("/", with: "."))"
-    }
-    return self.moduleName
-  }
-}
-
 /// The textual output of a module interface.
 public struct GeneratedInterfaceDetails: ResponseType, Hashable {
```

Comment:
When we remove the `OpenGeneratedInterfaceRequest`, we no longer need `GeneratedInterfaceDetails` in the `LanguageServerProtocol` module. I think `LanguageService.swift` would be a better location now.